lib/Logger.js

```diff
@@ -55,7 +77,9 @@ module.exports = class Logger {
 
   _output(level, record) {
     this._transports.forEach((transport) => {
-      transport(level, record);
+      const p = transport(level, record) || Promise.resolve();
```

Comment:
What happens if someone decides to return something from a transport which isn't null/undefined and isn't a promise either?

Reply:
Good point. I could test for a `then` property test the class